iFixit/checkout-ui-extensions
# Upgrade Shopify CLI, add checkout app permissions, update README

## QA

- The app should build with no errors using `pn dev` .
- The README should link to the correct locations.
- We can verify that the app is being used by the dev store by confirming that the "Codice Fiscale" field renders on the first page of checkout.
- If you try to view the app in Shopify Admin, nothing will render like in the screenshot below.

![image](https://github.com/iFixit/checkout-ui-extensions/assets/52104630/0992fe52-940b-4f13-8611-1fd2526a9dea)
https://ifixit-checkout-ui-extension-test.myshopify.com/admin/apps/ifixit-checkout-ui-extensions-2
